test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageSender.kt

```diff
@@ -115,3 +107,22 @@ class IntervalCoverageSender(
         }
     }
 }
+
+private fun processTotalSizeByte(logger: KLogger): BigInteger {
+    try {
+        val retentionLimit = JvmModuleConfiguration.getCoverageRetentionLimit()
```

Comment:
Most often, creating `val` too far off from their use site is unnecessary.
Declare it right when it's needed and see how the code changes

Reply:
changed